src/components/esn-app-grid/esn-app-grid.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface Item {
+  name: string;
+  iconUrl: string;
```

Comment:
It will not always be an URL. From what we defined, it may be an inlined image in base64 format

Reply:
Good to know that. I'll look into it. Is there any other possible format?

Reply:
check [here](https://ci.linagora.com/linagora/lgs/openpaas/esn/-/wikis/Frontend-Split/Application-Discovery-Service#icon)

the icon will be in this form:
```json
icon: {
  "type": "image/png",
  "data": "dGVzdAo="
}
```

Reply:
This won't be necessary as we will include all the icons for a fixed set of applications in the web component itself.